bot/tests/expected/expected_files/expected_list.txt

```diff
@@ -0,0 +1,31 @@
+🔍 Wyniki dla wyszukiwania: 'krowa' 🔍
```

Comment:
sure że to tu ma być? nie czaję po co to xD

Reply:
Nie xD zapodziało się ale dobrze że mówisz bo tu jest kolejne pytanie gdzie trzymać wzorce ? Tylko lokalnie na PC czy gdzieś to dać

Reply:
ja nawet nie wiem do czego to ma sluzyc i co to za "wzorzec"

Reply:
w skórcie muszę mieć jakieś expected filmik czy pliki do porównania z tym co wypluł bot ale tak teraz sobie myślę że wsm mógłbym trzymać same hashe i by stykło